spec/controllers/api/v2/pagination/pagination_spec_helper.rb

```diff
@@ -31,7 +31,7 @@ module V2
     module Pagination
       module PaginationSpecHelper
         def paginating_index_action(model, scope)
-          describe :index, type: :controller do
+          describe '#index', type: :controller do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.